requirements/build.txt

```diff
@@ -2,40 +2,40 @@
 # This file is autogenerated by pip-compile with Python 3.8
 # by the following command:
 #
-#    pip-compile --allow-unsafe --config=pyproject.toml requirements/build.in
```

Comment:
This seems to undo https://github.com/hypothesis/via/commit/20c3783fc530fcaf53381b823aad7c5582b71618

but it's mostly generated the same way :thinking: